routes/paper.py

```diff
@@ -219,18 +220,37 @@ def _get_comment(self, comment_id):
         if not comment:
             abort(404, messsage='Comment not found')
         return comment
-
+#################################
+#    def send_notification(to, user_1, user_2, paper_id, comment, reply):
+#        requests.post(
+#            "https://api.mailgun.net/v3/email.scihive.org/messages",
+#           auth=("api", "API KEY"),
+#            data={"from": "SciHive <mailgun@email.scihive.org>",
+#                  "to":[to],
+#                  "subject": "Reply to your comment on SciHive",
+#                  "text":"""Hello, {0}!\n{1} has replied to your comment on SchiHive:
+#                         \n{2}\n~{0}\n{3}\n~{1}\nClick here to view site.""".format(user_1, user_2, comment, reply),
+#                  "html":"""<p><b>Hello, {0}!</b><br>{1} has replied to your comment on SciHive:</p>
+#                         <p>{2}<br>~{0}</p><p>{3}<br>~{1}</p><br>
+#                         <a href='https://www.scihive.org/paper/{4}'>Click here to view site.</a>""".format(user_1, user_2, 
+#                                                                                                        comment, reply, paper_id)
+#             }
+#        )
+#################################
     @marshal_with(comment_fields, envelope='comment')
     def post(self, paper_id, comment_id):
         comment_id = {'_id': ObjectId(comment_id)}
-        self._get_comment(comment_id)
+        original_comment = self._get_comment(comment_id)
         data = new_reply_parser.parse_args()
         data['created_at'] = datetime.utcnow()
         data['id'] = str(uuid.uuid4())
         add_user_data(data)
         new_values = {"$push": {'replies': data}}
         db_comments.update_one(comment_id, new_values)
         comment = self._get_comment(comment_id)
+        #current_user = get_jwt_identity()
```

Comment:
Users can reply and comment anonymously (or even when they are not logged in), and we need to check all relevant scenarios